test/TestNullTasks.py

```diff
@@ -0,0 +1,10 @@
+SUCCESS_PLAYBOOK = '''
```

Comment:
Needs a docstring

Reply:
@webknjaz Are you joking? None of the constants used in our tests have docstrings attached to them and there is no need to have any. That kind of nitpicking assures that we fix bugs slower, delay releases and discourage contributions.

Reply:
I'm talking about the module docstrings